content/en/tracing/troubleshooting/connection_errors.md

```diff
@@ -0,0 +1,274 @@
+```
+
+{{< /programming-lang >}}
+
+{{< programming-lang lang=".NET" >}}
+
+#### Managed logs
+
+Even if debug mode isn’t enabled, the managed logs contain errors from connection refused issues:
+
+```
+{ MachineName: ".", Process: "[114 sample-web-app]", AppDomain: "[1 sample-web-app]", TracerVersion: "1.28.2.0" }
+2021-08-17 18:19:46.827 +00:00 [ERR] An error occurred while sending 1 traces to the agent at http://127.0.0.1:8126/v0.4/traces
+System.Net.Http.HttpRequestException: Connection refused
+ ---> System.Net.Sockets.SocketException (111): Connection refused
+   at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
+   --- End of inner exception stack trace ---
+
+```
+
+{{< /programming-lang >}}
+
+{{< programming-lang lang="php" >}}
+
+#### Tracer startup logs
+
+```
+Failed to connect to localhost port 8126: Connection refused
+```
+
+{{< /programming-lang >}}
+
+{{< programming-lang lang="cpp" >}}
+
+#### Application logs
+
+When the application cannot reach the Datadog Agent, these log messages are emitted to where your application sends logs:
+
+```
+Error sending traces to agent: Couldn't connect to server
+Failed to connect to localhost port 8126: Connection refused
+```
+
+
+{{< /programming-lang >}}
+
+{{< /programming-lang-wrapper >}}
+
+### Datadog Agent errors
+
+If the [`agent status`][3] command (available in Agent 6.20.0/7.20.0) shows that APM is not running or is not reachable on `localhost:8126`, your Datadog Agent does not have APM configured and traces cannot be submitted to the Datadog backend.
+
+```text
+APM Agent
+=========
+  Status: Not running or unreachable on localhost:8126.
+```
+
+## Troubleshooting the connection problem
+Whether it’s the tracing library or the Datadog Agent displaying the error, there are a few ways to troubleshoot.
+
+### Host-based setups
+
+If your application and the Datadog Agent are not containerized, the application with the tracing library should be trying to send traces to `localhost:8126` or `127.0.0.1:8126`, because that is where the Datadog Agent is listening. 
+
+If the Datadog Agent shows that APM is not listening, check for port conflicts with port 8126, which is what the APM component of the Datadog Agent uses by default.
+
+If you cannot isolate the root cause, [contact Datadog Support][4] with:
+- Information about the environment in which you’re deploying the application and Datadog Agent. 
+- If you are using proxies, information about how they’ve been configured.
+- If you are trying to change the default port from 8126 to another port, information about that port.
+- A [Datadog Agent flare][5].
+
+### Containerized setups
+
+#### Check network configuration
+
+In containerized setups, submitting traces to `localhost` or `127.0.0.1` is often incorrect since the Datadog Agent is also containerized and located elsewhere. **Note**: AWS ECS on Fargate is an exception to this rule.
```

Comment:
I just noticed right below it, it mentions not to set it for AWS EKS on Fargate too, so this should read:

**Note**: AWS ECS on Fargate and AWS EKS on Fargate are an exception to this rule

Reply:
```suggestion
In containerized setups, submitting traces to `localhost` or `127.0.0.1` is often incorrect since the Datadog Agent is also containerized and located elsewhere. **Note**: AWS ECS on Fargate and AWS EKS on Fargate are exceptions to this rule.
```